dvc/main.py

```diff
@@ -53,6 +54,14 @@ def main(argv=None):
         ret = 253
     except DvcParserError:
         ret = 254
+    except UnicodeError:
+        if not is_py2:
+            raise
```

Comment:
This will skip `except Exception` down below, so this is broken.

Reply:
@efiop , why? Sorry, but I'm not following :see_no_evil: 

I added a `raise Exception` in the `dvc add` command just to test it and it is returning the:

`ERROR: unexpected error` message

Reply:
ohhh, I think I get it; so it is a matter of displaying the `logger.exception("unexpected error")`

Reply:
it was not that intuitive to me, sorry about it, will make the change in a sec